### DISCUSSION TITLE
Create View through SQL Editor from two tables

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#8548] [Questions] [answered]
Trying to figure out the correct syntax for creating a view using the SQL Editor that:

selects A, B, C from TABLE1

and

selects D, E, F from TABLE2

and inserts both A, B, C and D, E, F as A, B, C in the view

Doable?

**Comment by MJDeligan** [comment, marked as answer]
Hey,

you should be able to alias D, E, F to A, B, C and the combine the two results with union like this:

```sql
CREATE VIEW <your_view_name_here> AS
SELECT A, B, C FROM TABLE1
UNION
SELECT D as A, E as B, F as C FROM TABLE2
```

This will give you a view with columns A, B, C where each row is either A, B, C from table1 or D, E, F from table2.

Is this what you want?

**Reply by ghost** [reply]
Thank you. Exactly :)
